src/pages/RequisitionsPage.js

```diff
@@ -31,7 +31,8 @@ export class RequisitionsPage extends React.Component {
       isRequestView: true,
       isCreatingRequisition: false,
     };
-    this.requisitions = props.database.objects('Requisition');
+    this.requisitions = props.database.objects('Requisition')
+                                      .filtered('serialNumber != "-1"');
```

Comment:
yes I agree, but, this doesn't help in deciphering differences in db wrappers:
```JS
    const database = new Database(schema);
    this.database = new UIDatabase(database);
    this.settings = new MobileAppSettings(this.database);
    migrateDataToVersion(this.database, this.settings);
    this.userAuthenticator = new UserAuthenticator(this.database, this.settings);
    const syncAuthenticator = new SyncAuthenticator(this.settings);
    this.synchroniser = new Synchroniser(database, syncAuthenticator, this.settings);
    this.postSyncProcessor = new PostSyncProcessor(this.database, this.settings);
```
A bit too messy in my opinion, this.database, database